search/src/test/scala/weco/api/search/works/WorksTest.scala

```diff
@@ -68,44 +58,83 @@ class WorksTest
   ) {
     withWorksApi {
       case (worksIndex, routes) =>
-        val works = indexedWorks(count = 3).sortBy {
-          _.state.canonicalId
-        }
+        indexTestDocuments(worksIndex, visibleWorks: _*)
 
-        insertIntoElasticsearch(worksIndex, works: _*)
+        val totalPages = math.ceil(visibleWorks.length / 2.0).toInt
 
-        assertJsonResponse(routes, s"$rootPath/works?page=2&pageSize=1") {
+        assertJsonResponse(routes, path = s"$rootPath/works?page=2&pageSize=2") {
```

Comment:
Not a change request, just a question: 

The order of pages being examined was 2,1,3. Now it's 2,3,1.

Is there a reason why it isn't 1,2,3?